### DISCUSSION TITLE
짱구 반성문

### DISCUSSION THREAD

**Discussion by 02ggang9** [BDD-CLUB/challenge-every-week-posting#539] [6. 반성문]
안녕하세요! 2주 연속 반성문 입니다 허허.. 

정말 죄송합니다 면접 준비한다고 아무 것도 손에 잡히지 않았네요.. 지금 면접 후기 작성하고 있는데 금방 올리도록 하겠습니다..!

**Comment by stopmin** [comment]
괜찮습니다 괜찮습니다
